### PR TITLE
 CRUD operation and Testing on Rating Model.

### PR DESCRIPTION
**Pull request by sefinehtesfa34**
CRUD operation and Testing on Rating Model.

### CODE REVIEW

File: starter-project-backend-g31/src/server.ts [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
+import dotenv from 'dotenv';
+import bodyParser from 'body-parser'
+dotenv.config();
 const PORT = process.env.PORT || 8000
 const DB_URI = process.env.MONGO_URI || "";
 
+app.use(bodyParser.urlencoded({ extended: true }))
+app.use(bodyParser.json())
 
+console.log("check: "+DB_URI);
```
Comment (RiyadHassen) [inline starter-project-backend-g31/src/server.ts:12]: remove log statments.

File: starter-project-backend-g31/src/server.ts [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
 mongoose.connect(DB_URI, {
     useNewUrlParser: true,
     useUnifiedTopology: true,
     useCreateIndex: true,
     useFindAndModify: false,
 })
 .then(() => {
-    app.listen(PORT, () => console.log('Server running...'));
+    console.log("connected to mongodb...")
```
Comment (RiyadHassen) [inline starter-project-backend-g31/src/server.ts:20]: same here

File: starter-project-backend-g31/src/controllers.ts [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            if (data) {
+                return data;
+            }
+            return {statusCode:404,message:"Request not found"}
+
+        })
+            
+};
+// Update rating
+export const update = (ratingId: string,json: { [x: string]: any; }) => {
```
Comment (RiyadHassen) [inline starter-project-backend-g31/src/controllers.ts:42]: accept req, res here from function parameters, and then do business processing over here.

**Comment by sefinehtesfa34** [comment]
updated pull request is needed

**Comment by sefinehtesfa34** [comment]
Ok cool

**Pull request closed without merging**